phil/cnx-routes.md

```diff
@@ -0,0 +1,302 @@
+# CNX APIs
+
+https://cnx.org APIs are split into 4 areas and documented in greater detail in each repository.
+
+You can retrieve published content using the [^cnx-archive] APIs (including search), and can change content using the [^cnx-publishing-api] APIs.
+
+Browser URLs are managed and documented in [^webview].
+
+## Identifiers
+
+There are 5 types of identifiers used within https://cnx.org
+
+- [sha hash](#sha-hash)
+- [UUID](#uuid)
+- [Short UUID](#short-uuid)
+- [UUID and version](#uuid-and-version)
```

Comment:
You'll likely want to make a not somewhere that throughout the code this is referred to as the 'ident-hash'.